frame/scored-pool/src/lib.rs

```diff
@@ -423,23 +436,27 @@ impl<T: Config<I>, I: 'static> Pallet<T, I> {
 	/// type function to invoke at the end of the method.
 	fn refresh_members(pool: PoolT<T, I>, notify: ChangeReceiver) {
 		let count = MemberCount::<T, I>::get();
+		let old_members = <Members<T, I>>::get();
+		let pool = pool.into_inner();
 
-		let mut new_members: Vec<T::AccountId> = pool
+		let new_members: Vec<T::AccountId> = pool
 			.into_iter()
 			.filter(|(_, score)| score.is_some())
 			.take(count as usize)
 			.map(|(account_id, _)| account_id)
 			.collect();
-		new_members.sort();
 
-		let old_members = <Members<T, I>>::get();
-		<Members<T, I>>::put(&new_members);
+		let mut new_members_bounded: MembersT<T, I> = BoundedVec::truncate_from(new_members);
```

Comment:
Please add a comment why truncating is fine here.

Reply:
I think its fine to use here if we add the integrity check that `MemberCount` does not exceed `MaximumMembers`.  
In any case; using `truncate_from` can silently ignore values, so its a good idea to add a comment in the code why it is being used.